### ISSUE TITLE
Add support for visionOS

### ISSUE THREAD

**Issue by jhrussell** [nalexn/ViewInspector#260]
Add support for using this project with visionOS.  

Currently v`0.9.8` has this error: 
```
Cannot find type 'LocationButton' in scope
```

**Comment by sisoje** [comment]
Hi, whats the status here? If you need help I can give it a try @nalexn

**Comment by nalexn** [comment]
Hey there, any help is appreciated and encouraged! I haven't look into this yet because visionOS requires beta Xcode plus there is likely more issues that I anticipate to arise when launching tests for visionOS, thus I was postponing looking into this until I have enough time.

**Comment by jhrussell** [comment]
Does `Package.swift` need to be updated as well?

```
// swift-tools-version:5.9
```

```
platforms: [
    .macOS(.v10_15), .iOS(.v11), .tvOS(.v13), .watchOS(.v7), .visionOS(.v1)
],
```

**Comment by nalexn** [comment]
Yes, but as of Xcode 15.2 SPM didn't yet add support for visionOS. If you open `Package.swift` in Xcode and CMD+click on `.watchOS`, for example, you'll see the factory methods defined for each platform except visionOS. The SDK has to have a `static func visionOS(_ version: PackageDescription.SupportedPlatform.VisionOSVersion) -> PackageDescription.SupportedPlatform`, but it's not there.

**Comment by jhrussell** [comment]
@nalexn Which tools version are you using for your package?  It has to be at least `5.9`: [SupportedPlatforms.swift](https://github.com/apple/swift-package-manager/blob/main/Sources/PackageDescription/SupportedPlatforms.swift#L237-L240)

**Comment by nalexn** [comment]
You are right, it was on `swift-tools-version:5.7`. Now it locates the factory. I'll get this sorted with the new release

**Comment by jhrussell** [comment]
Awesome.  Is there a tentative release schedule, yet?

**Comment by nalexn** [comment]
I'm hoping to wrap it up this weekend

**Comment by nalexn** [comment]
Released 0.9.10
